docsource/modules130-140.rst

```diff
@@ -20,7 +20,7 @@ Module coverage 13.0 -> 14.0
 +--------------------------------------------+-------------------------------------------------+
 |account_debit_note                          |                                                 |
 +--------------------------------------------+-------------------------------------------------+
-| |new| account_edi                          |                                                 |
+| |new| account_edi                          | Nothing to do                                   |
```

Comment:
This should be `Done`, as there's a migration script

Reply:
> This should be `Done`, as there's a migration script

@pedrobaeza i will fix it